ivy/functional/frontends/numpy/statistics/percentile.py

```diff
@@ -0,0 +1,27 @@
+# global
```

Comment:
Here we don't need a new file for percentile, we should implement this in order_statistics.py